Sources/Plasma/PubUtilLib/plFile/plSecureStream.cpp

```diff
@@ -95,6 +95,18 @@ fDeleteOnExit(false)
 
 plSecureStream::~plSecureStream()
 {
+    if (fOpenMode == kOpenWrite) {
+        fRAMStream->Rewind();
+        IWriteEncrypted(fRAMStream.get(), fWriteFileName);
+    }
+
+    if (fRef != INVALID_HANDLE_VALUE) {
+#if HS_BUILD_FOR_WIN32
+        CloseHandle(fRef);
+#elif HS_BUILD_FOR_UNIX
```

Comment:
```suggestion
#if defined(HS_BUILD_FOR_WIN32)
        CloseHandle(fRef);
#elif defined(HS_BUILD_FOR_UNIX)
```

Yeah, I see that it was wrong before 😞

Reply:
Will do. The codebase is quite inconsistent here though. `#if HS_BUILD_FOR_WIN32`, `#ifdef HS_BUILD_FOR_WIN32`, and `#if defined(HS_BUILD_FOR_WIN32)` are all relatively common.

Reply:
Yeah... Using `defined` here is more correct, though, because `hsConfig.h` is just doing `#define HS_BUILD_FOR_WIN32`, not `#define HS_BUILD_FOR_WIN32 1`, so we really need to check if the macro is defined, not its value. Unfortunately, `#elifdef` is only available in C++23 and later, but we're still on C++17.